### DISCUSSION TITLE
Using old fonts

### DISCUSSION THREAD

**Discussion by bingud** [excalidraw/excalidraw#8779] [Q&A] [answered]
Hello, i noticed when I made a new file that the fonts i used previously were replaced by only hand drawn fonts, and if i go back to the first file, the fonts are labeled `old`. Is there any way to use the old fonts in a new file?
Thanks

**Comment by Mrazator** [comment, marked as answer]
Yes! You can copy the elements with the old fonts to the new scene. This way they will appear in the fonts picker.

Also, in plus.excalidraw.com can choose to always show the old fonts. More info here: https://x.com/excalidraw/status/1816545056079147139?s=46

Could you give us a feedback why do you prefer the old fonts?

**Reply by bingud** [reply]
Thanks for the answer.
I prefer the old fonts because they looked more readable and also fit the style of documentation, i think more than the hand drawn style.
